test/e2e/testing-manifests/storage-csi/external-provisioner/rbac.yaml

```diff
@@ -61,6 +61,13 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
     verbs: ["get", "list", "watch"]
+  # (Alpha) Access to referencegrants is only needed when the CSI driver
+  # has the CrossNamespaceVolumeDataSource controller capability.
+  # In that case, external-provisioner requires "get", "list", "watch" 
+  # permissions  for "referencegrants" on "gateway.networking.k8s.io".
+  #- apiGroups: ["gateway.networking.k8s.io"]
+  #  resources: ["referencegrants"]
+  #  verbs: ["get", "list", "watch"]
```

Comment:
I'm not sure if we should add these entries here.  This is an alpha feature and it is commented out anyway.  If anyone wants to test this feature, they can just add these themselves.

Reply:
this file is synced from https://github.com/kubernetes-csi/external-provisioner/blob/v3.6.3/deploy/kubernetes/rbac.yaml#L60 via [update-hostpath script](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/testing-manifests/storage-csi/update-hostpath.sh#L90)